### PR TITLE
restructured components folder

### PR DESCRIPTION
**Pull request by BeeSeeWhy**
issue #158

### CODE REVIEW

File: src/components/personTable/PersonTable.tsx
Comment (jonahchoi) [inline src/components/personTable/PersonTable.tsx:1]: This file can be with the rest of the table files. We can rename this one to just 'table.tsx'.

Comment (BeeSeeWhy) [reply src/components/personTable/PersonTable.tsx:1]: I tried renaming to table.tsx and moving to table folder, but it kept giving me an error saying that it couldn't find "@/styles/Button.module.css" in the imports. If I left as personTable, everything was ok.

Comment (jonahchoi) [reply src/components/personTable/PersonTable.tsx:1]: I think the issue is with table.ts having the same name. For now, can you rename `table.ts` -> `tableTypes.ts`, and `PersonalTable.tsx` -> `table.tsx`? We might end up removing that types file in the future anyway.

**Review by jonahchoi — APPROVED** [review]
Looks good

**Pull request merged**